### ISSUE TITLE
how to use get_fields ?

### ISSUE THREAD

**Issue by alanotmt** [facebookresearch/detectron2#1067]
i know how to get pred_masks. 
i modifed pred_masks.

i tried this:
mask= outputs["instances"].get("pred_mask")
mask= torch.cat([mask[0:1],mask[2:]]) 

now how to replace to dict['instances']?

i read structures.instances ,
and find the  'get_fields'  is  'Modifying the returned dict will modify this instance.'

but i don't know how to write in code. Can you give  me a example?

i tried this:
outputs["instances"].get_fields

i am sorry about my fool

thank you

**Comment by ppwwyyxx** [comment]
`outputs["instances"].pred_masks = new_mask`

See "basic usage" in https://detectron2.readthedocs.io/modules/structures.html#detectron2.structures.Instances
